modules/serverpod_auth/serverpod_auth_server/test_integration/signout/signout_lagacy_option_test.dart

```diff
@@ -0,0 +1,217 @@
+import 'package:serverpod/serverpod.dart';
+import 'package:serverpod_auth_server/serverpod_auth_server.dart';
+import 'package:test/test.dart';
+import '../../test/integration/test_tools/serverpod_test_tools.dart';
+
+void main() {
+  withServerpod(
+    'Given different legacy sign-out options',
```

Comment:
Suggestion: Place the different options in different `withServerpod`.

then you will get
`Given no legacy user sign-out option defined and user signed in to multiple devices when signing out user then user is signed out of all devices`.

Break the tests into fitting groups to achive this.